### ISSUE TITLE
CupertinoPicker skip items in Desktop

### ISSUE THREAD

**Issue by deczaloth** [flutter/flutter#119500]
## Steps to Reproduce

1. Run the code sample to deploy in Windows(Desktop)
2. Do simple scrolling with the mouse.

**Expected results:**
Being able to scroll to any of the displayed items.

**Actual results:**
Picker skips items

<details>
<summary>Code sample</summary>

```dart
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: const MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({super.key, required this.title});

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {

  int _index = 0;

  List<String> team = <String>["Olaf","Victor","Rita"] ;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child: CupertinoPicker(
            itemExtent: 50,
            onSelectedItemChanged: (selectedIndex) {
              _index = selectedIndex;
            },
            children: List.generate(team.length, (index) {
              return Text(team[index]);
            }),
        ),
      ),
    );
  }
}
```

</details>

<details>
  <summary>Logs</summary>

```
[  +22 ms] executing: [C:\src\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +109 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms] b06b8b2710955028a6b562f5aa6fe62941d6febf                                                 
[        ] executing: [C:\src\flutter/] git tag --points-at b06b8b2710955028a6b562f5aa6fe62941d6febf
[ +111 ms] Exit code 0 from: git tag --points-at b06b8b2710955028a6b562f5aa6fe62941d6febf
[        ] 3.7.0
[  +16 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref --symbolic @{upstream}
[  +92 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{upstream}
[        ] origin/stable
[   +1 ms] executing: [C:\src\flutter/] git ls-remote --get-url origin
[  +90 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +97 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref HEAD
[  +89 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +76 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +86 ms] executing: C:\Users\Julian\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +95 ms] List of devices attached
[   +9 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +5 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] [1]: Windows (windows)
[        ] [2]: Chrome (chrome)
[        ] [3]: Edge (edge)
[   +2 ms] Please choose one (To quit, press "q/Q")
[        ] :
[+60466 ms] 1
[  +22 ms] Skipping pub get: version match.
[ +213 ms] Generating C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +116 ms] Initializing file store
[  +18 ms] Skipping target: gen_localizations
[  +12 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents:
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\.dart_tool\package_config_subset}                                      
[  +43 ms] gen_dart_plugin_registrant: Complete
[   +3 ms] Skipping target: _composite
[   +3 ms] complete                   
[   +8 ms] Launching lib\main.dart on Windows in debug mode...
[   +6 ms] C:\src\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\src\flutter\bin\cache\dart-sdk\bin\snapshots\frontend_server.dart.snapshot --sdk-root
C:\src\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill    
C:\Users\Julian\AppData\Local\Temp\flutter_tools.57a6f846\flutter_tool.5a32c15e\app.dill --packages
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation 
--filesystem-scheme org-dartlang-root --initialize-from-dill build\c075001b96339384a97db4862b8ab8db.cache.dill.track.dill --verbosity=error
--enable-experiment=alternative-invalidation-strategy
[  +33 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires
Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[  +73 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires
Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[   +1 ms] [
             {
               "instanceId": "a2bafe6d",
               "installDate": "2022-01-14T12:19:55Z",
               "installationName": "VisualStudio/17.4.2+33122.133",
               "installationPath": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional",
               "installationVersion": "17.4.33122.133",
               "productId": "Microsoft.VisualStudio.Product.Professional",
               "productPath": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Professional 2022",
               "description": "Professionelle IDE, optimal für kleine Teams",
               "channelId": "VisualStudio.17.Release",
               "channelUri": "https://aka.ms/vs/17/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "installedChannelId": "VisualStudio.17.Release",
               "installedChannelUri": "https://aka.ms/vs/17/release/channel",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.4#17.4.2",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=661288",
               "updateDate": "2022-12-12T10:07:24.0697588Z",
               "catalog": {
                 "buildBranch": "d17.4",
                 "buildVersion": "17.4.33122.133",
                 "id": "VisualStudio/17.4.2+33122.133",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "17.4.2",
                 "productLine": "Dev17",
                 "productLineVersion": "2022",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "2",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "17.4.2+33122.133",
                 "requiredEngineVersion": "3.4.2244.14676"
               },
               "properties": {
                 "campaignId": "2016712300.1479193388",
                 "channelManifestId": "VisualStudio.17.Release/17.4.2+33122.133",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +6 ms] Building Windows application...
[  +24 ms] <- compile package:cupertinopicker/main.dart
[   +2 ms] executing: C:\Program Files\Microsoft Visual Studio\2022\Professional\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\windows -B build\windows -G Visual Studio 17 2022
[ +127 ms] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19045.
[  +48 ms] -- Configuring done
[ +331 ms] -- Generating done
[  +32 ms] -- Build files have been written to: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows
[  +11 ms] executing: C:\Program Files\Microsoft Visual Studio\2022\Professional\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug
--target INSTALL --verbose
[ +244 ms] MSBuild version 17.4.0+18d5aef85 for .NET Framework
[  +84 ms] Build started 30.01.2023 10:55:54.
[ +373 ms] Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[   +1 ms] Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\INSTALL.vcxproj" (1) is building
"C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +34 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +93 ms]   Checking Build System
[ +101 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +3 ms] Done Building Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +176 ms] Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\INSTALL.vcxproj" (1) is building
"C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[   +1 ms] Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\runner\cupertinopicker.vcxproj" (4) on node 1 (default targets).
[   +1 ms] Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\runner\cupertinopicker.vcxproj" (4) is building
"C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +10 ms] CustomBuild:
[        ]   "The build of 'C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\CMakeFiles\950e9fb01cd50287143050d87e23a745\flutter_assemble.rule' depends on 
'C:\USERS\JULIAN\DECZALOTH\ANDROIDSTUDIOPROJECTS\CUPERTINOPICKER\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\CMakeFiles\3f64e8eaa790657894e76e79b8bcedad\flutter_windows.dll.rule'. The items cannot be built in      
parallel."
[ +120 ms]   Generating C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/flutter_windows.dll,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/flutter_export.h,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/flutter_windows.h,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/flutter_messenger.h,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/flutter_plugin_registrar.h,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/flutter_texture_registrar.h,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc,
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[ +737 ms]   [  +20 ms] executing: [C:\src\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +121 ms]   [ +150 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ]   [   +1 ms] b06b8b2710955028a6b562f5aa6fe62941d6febf
[        ]   [        ] executing: [C:\src\flutter/] git tag --points-at b06b8b2710955028a6b562f5aa6fe62941d6febf
[ +141 ms]   [ +142 ms] Exit code 0 from: git tag --points-at b06b8b2710955028a6b562f5aa6fe62941d6febf
[        ]   [        ] 3.7.0
[  +21 ms]   [  +22 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref HEAD
[ +105 ms]   [ +105 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms]   [        ] stable
[  +74 ms]   [  +75 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +2 ms]   [   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +151 ms]   [ +154 ms] executing: [C:\src\flutter/] git rev-parse --abbrev-ref --symbolic @{upstream}
[ +123 ms]   [ +123 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{upstream}
[        ]   [        ] origin/stable
[        ]   [        ] executing: [C:\src\flutter/] git ls-remote --get-url origin
[ +107 ms]   [ +108 ms] Exit code 0 from: git ls-remote --get-url origin
[        ]   [        ] https://github.com/flutter/flutter.git
[        ]   [   +1 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +4 ms]   [   +4 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +57 ms]   [  +65 ms] Initializing file store
[  +32 ms]   [  +32 ms] Skipping target: gen_localizations
[  +22 ms]   [  +21 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents:
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\.dart_tool\package_config_subset}
[   +7 ms]   [   +7 ms] unpack_windows: Starting due to {}
[ +614 ms]   [ +613 ms] unpack_windows: Complete
[+5285 ms]   [+5286 ms] gen_dart_plugin_registrant: Complete
[   +3 ms]   [   +3 ms] kernel_snapshot: Starting due to {}
[  +13 ms]   [  +13 ms] C:\src\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\src\flutter\bin\cache\dart-sdk\bin\snapshots\frontend_server.dart.snapshot --sdk-root
C:\src\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --target=flutter --no-print-incremental-dependencies -DFLUTTER_WEB_AUTO_DETECT=true -Ddart.vm.profile=false       
-Ddart.vm.product=false --enable-asserts --track-widget-creation --packages C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\.dart_tool\package_config.json --output-dill
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\.dart_tool\flutter_build\7f49c8caec9f669c849251c5c96d5ba3\app.dill --depfile
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\.dart_tool\flutter_build\7f49c8caec9f669c849251c5c96d5ba3\kernel_snapshot.d --incremental --initialize-from-dill        
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\.dart_tool\flutter_build\7f49c8caec9f669c849251c5c96d5ba3\app.dill --verbosity=error package:cupertinopicker/main.dart  
[+11219 ms]   [+11219 ms] kernel_snapshot: Complete
[ +584 ms]   [ +584 ms] debug_bundle_windows_assets: Starting due to {}
[ +122 ms]   [ +122 ms] shaderc command: [C:\src\flutter\bin\cache\artifacts\engine\windows-x64\impellerc.exe, --sksl, --iplr,
--sl=C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\flutter_assets\shaders/ink_sparkle.frag,
--spirv=C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\flutter_assets\shaders/ink_sparkle.frag.spirv,
--input=C:\src\flutter\packages\flutter\lib\src\material\shaders\ink_sparkle.frag, --input-type=frag, --remap-samplers,
--include=C:\src\flutter\packages\flutter\lib\src\material\shaders, --include=C:\src\flutter\bin\cache\artifacts\engine\windows-x64\shader_lib]
[ +373 ms]   [ +372 ms] debug_bundle_windows_assets: Complete
[ +580 ms]   [ +581 ms] Persisting file store
[  +12 ms]   [  +12 ms] Done persisting file store
[  +11 ms]   [  +11 ms] build succeeded.
[   +9 ms]   [   +9 ms] "flutter assemble" took 19.533ms.
[  +94 ms]   [  +94 ms] ensureAnalyticsSent: 90ms
[        ]   [        ] Running 0 shutdown hooks
[        ]   [        ] Shutdown hooks complete
[        ]   [        ] exiting with code 0
[ +104 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[   +2 ms] Done Building Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[  +43 ms] Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\runner\cupertinopicker.vcxproj" (4) is building
"C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\flutter\flutter_wrapper_app.vcxproj" (6) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_app.dir\Debug\flutter_.5DA57948.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +75 ms] ClCompile:
[   +1 ms]   All outputs are up-to-date.
[  +38 ms] Lib:
[   +1 ms]   All outputs are up-to-date.
[   +2 ms]   flutter_wrapper_app.vcxproj -> C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\flutter\Debug\flutter_wrapper_app.lib
[  +22 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.5DA57948.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_app.dir\Debug\flutter_.5DA57948.tlog\flutter_wrapper_app.lastbuildstate".
[        ] Done Building Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[  +12 ms] InitializeBuildStatus:
[        ]   Creating "cupertinopicker.dir\Debug\cupertinopicker.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +7 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +29 ms] ClCompile:
[        ]   C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.34.31933\bin\HostX64\x64\CL.exe /c
/IC:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\windows /IC:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\windows\flutter\ephemeral
/IC:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\windows\flutter\ephemeral\cpp_client_wrapper\include /Zi /nologo /W4 /WX /diagnostics:column /Od /Ob0 /D _UNICODE /D  
UNICODE /D WIN32 /D _WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D "FLUTTER_VERSION=\"1.0.0+1\"" /D FLUTTER_VERSION_MAJOR=1 /D FLUTTER_VERSION_MINOR=0 /D FLUTTER_VERSION_PATCH=0 /D        
FLUTTER_VERSION_BUILD=1 /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /Gm- /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17
/Fo"cupertinopicker.dir\Debug\\" /Fd"cupertinopicker.dir\Debug\vc143.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\windows\flutter\generated_plugin_registrant.cc
C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\windows\runner\flutter_window.cpp
[  +94 ms]   generated_plugin_registrant.cc
[ +530 ms]   flutter_window.cpp
[+1115 ms]   Generating Code...
[ +119 ms] ResourceCompile:
[   +1 ms]   All outputs are up-to-date.
[  +50 ms] Link:
[   +1 ms]   C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.34.31933\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE
/OUT:"C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\runner\Debug\cupertinopicker.exe" /INCREMENTAL /ILK:"cupertinopicker.dir\Debug\cupertinopicker.ilk" 
/NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib dwmapi.lib C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\windows\flutter\ephemeral\flutter_windows.dll.lib
kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'"        
/manifest:embed /manifestinput:C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\windows\runner\runner.exe.manifest /DEBUG
/PDB:"C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/cupertinopicker.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT
/IMPLIB:"C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/cupertinopicker.lib" /MACHINE:X64  /machine:x64 cupertinopicker.dir\Debug\Runner.res
[   +2 ms]   cupertinopicker.dir\Debug\flutter_window.obj
[        ]   cupertinopicker.dir\Debug\main.obj
[        ]   cupertinopicker.dir\Debug\utils.obj
[        ]   cupertinopicker.dir\Debug\win32_window.obj
[        ]   cupertinopicker.dir\Debug\generated_plugin_registrant.obj
[ +289 ms]   cupertinopicker.vcxproj -> C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\runner\Debug\cupertinopicker.exe
[  +15 ms] FinalizeBuildStatus:
[        ]   Deleting file "cupertinopicker.dir\Debug\cupertinopicker.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "cupertinopicker.dir\Debug\cupertinopicker.tlog\cupertinopicker.lastbuildstate".
[        ] Done Building Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\runner\cupertinopicker.vcxproj" (default targets).
[  +49 ms] Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (7) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.432115F1.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +13 ms] ClCompile:
[   +1 ms]   All outputs are up-to-date.
[   +4 ms] Lib:
[   +1 ms]   All outputs are up-to-date.
[   +1 ms]   flutter_wrapper_plugin.vcxproj -> C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[  +12 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.432115F1.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.432115F1.tlog\flutter_wrapper_plugin.lastbuildstate".
[        ] Done Building Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[  +14 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +3 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +57 ms]   Building Custom Rule C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/windows/CMakeLists.txt
[  +69 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[   +3 ms] Done Building Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +9 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +75 ms] PostBuildEvent:
[   +1 ms]   setlocal
[        ]   "C:\Program Files\Microsoft Visual Studio\2022\Professional\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P cmake_install.cmake    
[        ]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[  +73 ms]   -- Install configuration: "Debug"
[  +28 ms]   -- Up-to-date: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/icudtl.dat
[   +1 ms]   -- Up-to-date: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/flutter_windows.dll
[  +10 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets
[   +1 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[   +2 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[   +2 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/fonts
[   +2 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +8 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[ +132 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/NOTICES.Z
[   +3 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/packages
[   +1 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[   +1 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[   +1 ms]   -- Installing:
C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[   +3 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/shaders
[   +2 ms]   -- Installing: C:/Users/Julian/deczaloth/AndroidStudioProjects/cupertinopicker/build/windows/runner/Debug/data/flutter_assets/shaders/ink_sparkle.frag
[  +20 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[   +4 ms] Done Building Project "C:\Users\Julian\deczaloth\AndroidStudioProjects\cupertinopicker\build\windows\INSTALL.vcxproj" (default targets).
[   +8 ms] Build succeeded.
[        ]     0 Warning(s)
[   +1 ms]     0 Error(s)
[        ] Time Elapsed 00:00:24.82
[  +26 ms] Building Windows application... (completed in 25,7s)
[ +226 ms] Observatory URL on device: http://127.0.0.1:57043/eyW1pEvIFqA=/
[   +4 ms] Caching compiled dill
[  +30 ms] Connecting to service protocol: http://127.0.0.1:57043/eyW1pEvIFqA=/
[ +166 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:57043/eyW1pEvIFqA=/.
[ +138 ms] DDS is listening at http://127.0.0.1:57048/zhPUKGcNxZQ=/.
[ +109 ms] Successfully connected to service protocol: http://127.0.0.1:57043/eyW1pEvIFqA=/
[  +43 ms] DevFS: Creating new filesystem on the device (null)
[  +37 ms] DevFS: Created new filesystem on the device (file:///C:/Users/Julian/AppData/Local/Temp/cupertinopicker2856925/cupertinopicker/)
[  +10 ms] Updating assets
[ +166 ms] Syncing files to device Windows...
[   +9 ms] Compiling dart to kernel with 0 updated files
[   +1 ms] Processing bundle.
[   +5 ms] <- recompile package:cupertinopicker/main.dart 48f96ce3-cb7f-4c8b-99fd-048fba2dbdfa
[   +2 ms] <- 48f96ce3-cb7f-4c8b-99fd-048fba2dbdfa
[   +9 ms] Bundle processing done.
[ +292 ms] Updating files.
[  +29 ms] DevFS: Sync finished
[   +2 ms] Syncing files to device Windows... (completed in 348ms)
[   +1 ms] Synced 0.0MB.
[   +2 ms] <- accept
[   +8 ms] Connected to _flutterView/0x2891539be40.
[   +9 ms] Flutter run key commands.
[   +8 ms] r Hot reload. 
[   +3 ms] R Hot restart.
[   +2 ms] h List all available interactive commands.
[   +3 ms] d Detach (terminate "flutter run" but leave application running).
[   +1 ms] c Clear the screen
[   +1 ms] q Quit (terminate the application on the device).
[        ]  Running with sound null safety 
[   +2 ms] An Observatory debugger and profiler on Windows is available at: http://127.0.0.1:57048/zhPUKGcNxZQ=/
[ +157 ms] The Flutter DevTools debugger and profiler on Windows is available at: http://127.0.0.1:9106?uri=http://127.0.0.1:57048/zhPUKGcNxZQ=/
```

```
Analyzing cupertinopicker...                                            

   info - The value of the field '_index' isn't used - lib\main.dart:34:7 - unused_field

1 issue found. (ran in 2.7s)
```

```
[√] Flutter (Channel stable, 3.7.0, on Microsoft Windows [Version 10.0.19045.2486], locale de-DE)
    • Flutter version 3.7.0 on channel stable at C:\src\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b06b8b2710 (6 days ago), 2023-01-23 16:55:55 -0800
    • Engine revision b24591ed32
    • Dart version 2.19.0
    • DevTools version 2.20.1

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
    • Android SDK at C:\Users\Julian\AppData\Local\Android\sdk
    • Platform android-33, build-tools 33.0.1
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Professional 2022 17.4.2)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Professional
    • Visual Studio Professional 2022 version 17.4.33122.133
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2022.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19045.2486]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 109.0.5414.76
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 109.0.1518.70

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```

</details>

**Comment by deczaloth** [comment]
A workaround has been shared in this StackOverflow answer:

https://stackoverflow.com/questions/75282825/cupertinopicker-skip-items-in-desktop-windows-in-flutter-3-7

**Comment by exaby73** [comment]
## Triage report

I can reproduce this issue with stable and master. I was only able to reproduce this with a mouse and not a trackpad. Unfortunately, I cannot confirm if this only occurs on windows, since my MacBook Air does not have a USB A connection and my dongle recently has stopped working.

This issue seems related to https://github.com/flutter/flutter/issues/119122. The mouse jumps multiple lines. But whether or not this mouse jumping many lines is a bug or not, the user can always change mouse settings to scroll 10 lines for example, which means that no matter if the scrolling is fixed or not, this will continue to be an issue on a few edge cases. Therefore, I think this can rather be a proposal to add a way for the scroll to snap to each item once scrolling is finished. I'll label it as a proposal but I would love insights from the team as to if this is a bug rather than a proposal.

<details><summary><code>flutter doctor -v</code> (Master)</summary>

```
[!] Flutter (Channel master, 3.7.0-30.0.pre.7, on Microsoft Windows [Version 10.0.22621.1105], locale en-US)
    • Flutter version 3.7.0-30.0.pre.7 on channel master at C:\Users\Nabeel\fvm\versions\master
    ! The flutter binary is not on your path. Consider adding C:\Users\Nabeel\fvm\versions\master\bin to your path.
    ! Warning: `dart` on your path resolves to C:\tools\dart-sdk\bin\dart.exe, which is not inside your current Flutter SDK checkout at C:\Users\Nabeel\fvm\versions\master. Consider adding C:\Users\Nabeel\fvm\versions\master\bin to the front of your path.
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 15cd00f1ed (6 hours ago), 2023-01-29 21:58:05 -0500
    • Engine revision a7bb0e4108
    • Dart version 3.0.0 (build 3.0.0-179.0.dev)
    • DevTools version 2.20.1
    • If those were intentional, you can disregard the above warnings; however it is recommended to use "git" directly to perform update checks and upgrades.

[✓] Windows Version (Installed version of Windows is version 10 or higher)

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at C:\android-sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: C:\Users\Nabeel\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\213.7172.25.2113.9014738\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = C:\Program Files\BraveSoftware\Brave-Browser\Application\brave.exe

[✓] Visual Studio - develop for Windows (Visual Studio Community 2022 17.4.2)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.4.33122.133
    • Windows 10 SDK version 10.0.22621.0

[✓] Android Studio (version 2021.3)
    • Android Studio at C:\Users\Nabeel\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\213.7172.25.2113.9014738
    • Flutter plugin version 70.2.3
    • Dart plugin version 213.7433
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)

[✓] VS Code (version 1.74.3)
    • VS Code at C:\Users\Nabeel\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.46.0

[✓] VS Code (version 1.73.0-insider)
    • VS Code at C:\Users\Nabeel\AppData\Local\Programs\Microsoft VS Code Insiders
    • Flutter extension version 3.50.0

[✓] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22621.1105]
    • Chrome (web)      • chrome  • web-javascript • unknown
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 109.0.1518.52

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```

</details>

PS: @deczaloth your workaround in Stackoverflow; you should place the initialization of the ScrollController and `addListener` in `initState` and dispose of it in `dispose`. They way you have done it currently, would cause a memory leak and performance problems

**Comment by deczaloth** [comment]
> 

Thanks for the suggestion. I made the changes in[ SO](https://stackoverflow.com/questions/75282825/cupertinopicker-skip-items-in-desktop-windows-in-flutter-3-7).

**Comment by victorsanni** [comment]
Closing as duplicate of https://github.com/flutter/flutter/issues/88954.
